platform/commonUI/browse/src/windowing/NewTabAction.js

```diff
@@ -30,23 +30,25 @@ define(
         "use strict";
 
         /**
-         * The new window action allows a domain object to be opened
-         * into a new browser window. (Currently this is a stub, present
+         * The new tab action allows a domain object to be opened
+         * into a new browser tab. (Currently this is a stub, present
          * to allow the control to appear in the appropriate location in
          * the user interface.)
          * @constructor
          */
-        function NewWindowAction($window) {
+        function NewTabAction($window) {
             return {
                 /**
-                 * Open the object in a new window (currently a stub)
+                 * Open the object in a new tab
                  */
                 perform: function () {
-                    $window.alert("Not yet functional. This will open objects in a new window.");
+                    // Places the current Url into a variable
+                    // Then the Url is opened using $window
+                    $window.open($window.location.href);
```

Comment:
Couple of notes here:

* Instead of relying on the current URL being correct, we want to generate the URL based on the path to the domain object. See 
[BrowseController](https://github.com/nasa/openmctweb/blob/ee69eb3a01a2bc63ab65cad29f090dd517a15458/platform/commonUI/browse/src/BrowseController.js#L50-L65) for an example of this. This will allow the Open in New Tab action to work on objects that are right-clicked on (but not necessarily navigated-to)
* Adding a second argument of `"_blank"` will get this to behave more consistently between browsers